site-src/guides/tls.md

```diff
@@ -25,30 +25,33 @@ upstream connections is managed independently.
 
 Depending on the Listener Protocol, different TLS modes and Route types are supported.
 
-Listener Protocol | TLS Mode | Route Type Supported
---- | --- | ---
-TLS | Passthrough | TLSRoute
-TLS | Terminate | TCPRoute
-HTTPS | Terminate | HTTPRoute
+| Listener Protocol | TLS Mode    | Connections Affected   | Route Type Supported |
+|-------------------|-------------|-------------------------|----------------------|
+| TLS               | Passthrough | Downstream and Upstream | TLSRoute             |
+| TLS               | Terminate   | Downstream              | TCPRoute             |
+| HTTPS             | Terminate   | Downstream              | HTTPRoute            |
+| HTTPS             | Backend     | Upstream                | HTTPRoute            |
 
 Please note that in case of `Passthrough` TLS mode, no TLS settings take
-effect as the TLS session from the client is NOT terminated at the Gateway.
-The rest of the document assumes that TLS is being terminated at the Gateway,
-which is the default setting.
+effect as the TLS session from the client is NOT terminated at the Gateway, but rather
+passes through the Gateway, encrypted.
+In the case of `Backend`, only the Upstream connection is affected.
+For `HTTPRoute`, the use of both `Terminate` and `Backend` is supported, and provides what
+is commonly known as a connection that is terminated and then re-encrypted at the Gateway.
```

Comment:
Similar comment as above, I think it's best to revert this change since `Backend` is not really the same thing as `Terminate` or `Passthrough`.

Reply:
Done.